src/render/components/survey.js

```diff
@@ -0,0 +1,235 @@
+const {html, heading, slugify} = require("./bits");
+
+const experience = (id, title) => {
+  return html`
+    <fieldset>
+      <legend>${title}</legend>
+      <div class="radio-options">
+        <div class="radio-option">
+          <input type="radio" id="${id + "-0"}" name="${id}" value="unknown">
+          <label for="${id + "-0"}">What's that?</label>
+        </div>
+        <div class="radio-option">
+          <input type="radio" id="${id + "-1"}" name="${id}" value="uninterested">
+          <label for="${id + "-1"}">Uninterested for now</label>
+        </div>
+        <div class="radio-option">
+          <input type="radio" id="${id + "-2"}" name="${id}" value="curious">
+          <label for="${id + "-2"}">Curious about</label>
+        </div>
+        <div class="radio-option">
+          <input type="radio" id="${id + "-3"}" name="${id}" value="need-help">
+          <label for="${id + "-3"}">I would like more info</label>
+        </div>
+        <div class="radio-option">
+          <input type="radio" id="${id + "-4"}" name="${id}" value="easy">
+          <label for="${id + "-4"}">I know how to do this</label>
+        </div>
+      </div>
+    </fieldset>
+  `;
+};
+
+const checkAny = (id, title, opts) => {
+  return html`
+    <fieldset>
+      <legend>${title} <small><em>(Check all that apply)</em></small></legend>
+      ${opts.map(({label, id: optId}) => {
+        const optIdJoined = id + "-" + optId;
+        return html`
+          <input type="checkbox" id="${optIdJoined}" name="${optIdJoined}">
+          <label for="${optIdJoined}">${label}</label>
+          <br>
+        `;
+      })}
+    </fieldset>
+  `;
+};
+
+const headings = [];
+
+const section = (level, title) => {
+  headings.push({title, id: slugify(title), level});
+  return heading("h" + level, title);
+};
+
+const body = html`
+  <form method="post" action="http://api.reclaimers.net:8080/survey/submit" enctype="multipart/form-data">
```

Comment:
Note here that I've hardcoded the form to submit to `api.reclaimers.net:8080` which is the EC2 instance running this branch. If I just did the relative URL `/survey/submit`, POSTs would be made against the CDN and its origin (an S3 bucket of static files) is not capable of handling such requests.